omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ObsController1_9Test.java

```diff
@@ -258,6 +258,24 @@ public void getObs_shouldCreateAnObsWhenTheQuestionConceptIsSetAsAMapContainingT
 		assertEquals(++originalCount, getAllCount());
 	}
 	
+	@Test
+	public void shouldCreateObsAWithADateTimeValueHavingATimeZone() throws Exception {
+		
+		executeDataSet("customConceptDataset1_9.xml");
+		long originalCount = getAllCount();
+		SimpleObject obs = new SimpleObject();
+		obs.add("person", "da7f524f-27ce-4bb2-86d6-6d1d05312bd5");
+		SimpleObject concept = new SimpleObject();
+		concept.add("uuid", "A607c80f-1ea9-4da3-bb88-6276ce8868ii");
+		concept.add("name", new SimpleObject());
+		obs.add("concept", concept);
+		obs.add("obsDatetime", "2018-11-13T00:00:00.000-0500");
+		obs.add("value", "2018-11-13T00:00:06.040-0500");
+		String json = new ObjectMapper().writeValueAsString(obs);
+		handle(newPostRequest(getURI(), json));
+		assertEquals(++originalCount, getAllCount());	
```

Comment:
@mozzy11 , can we add more meaningful asserts to this test?

Reply:
well , so far what was more impontant was to test that a new a obs  of dateTime value can be added , when the date contains a time zone parameter. 

let me see what other imporntant feature can be relevant to test

Reply:
i also added some other assert